test/generate-json.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import time
+import os
+import struct
+
+# PingMessage is generated by this project
+import sys
```

Comment:
nice hack

Reply:
should we export it before running the script?

Reply:
the run-tests.sh will deal with everything for you, if you want to run all tests.

Reply:
why is this hack in here then.

Reply:
you don't need to export ping-python generated files for portability, run-tests will generate and run this script.
This line is necessary to import PingMessage from a different path.